core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -160,6 +173,43 @@ export class WorkflowEditorComponent implements AfterViewInit {
     }
   }
 
+  private _unregisterKeyboard() {
+    document.removeEventListener("keydown", this._handleKeyboardAction.bind(this));
+  }
+
+  private _registerKeyboard() {
+    document.addEventListener("keydown", this._handleKeyboardAction.bind(this));
+  }
+
+  private _handleKeyboardAction(event: any) {
+    this._onProcessKeyboardActionObservable = new Subject();
+    event.preventDefault();
+    this.workflowVersionService
```

Comment:
Why is this workflowversion preview in this logic?

Reply:
This is listening for when workflow version preview is opened, do not enable undo/redo hotkey.